integrationTests/i3CRVlvUSDCurvePool.ts

```diff
@@ -0,0 +1,82 @@
+// We import Chai to use its asserting functions here.
+import { expect } from "chai";
+import { Contract, Signer } from "ethers";
+import { ethers } from "hardhat";
+import { abilvUSD, abilvUSD3CRVPool, abiUSDC, abiZap } from "../test/ABIs";
+import { addressZap, address3CRV, addressUSDC, defaultBlockNumber, helperResetNetwork, address3CRVlvUSDPool } from "../test/MainnetHelper";
+import { impersonateAccount, fundAccount, stopImpersonate, addresslvUSDToken } from "./IntegrationTestContext";
+
+const About4700 = "0x1000000000000000000";
+
+const addr1 = "0x55fe002aeff02f77364de339a1292923a15844b8"; // Circles address
```

Comment:
What is circles?

Reply:
Forgot the ' - it's Circle's (Coinbase and Circle are the main companies behind USDC)